addons/project/models/project.py

```diff
@@ -74,32 +74,36 @@ def _get_default_project_ids(self):
         return [default_project_id] if default_project_id else None
 
     active = fields.Boolean('Active', default=True)
-    name = fields.Char(string='Name', required=True, translate=True)
+    name = fields.Char(string='Name', required=True, translate=True,
+        help="Name of your stage e.g. To do")
     description = fields.Text(translate=True)
-    sequence = fields.Integer(default=1)
+    sequence = fields.Integer(default=1,
+        help="Determines the order in which stages are displayed.")
     project_ids = fields.Many2many('project.project', 'project_task_type_rel', 'type_id', 'project_id', string='Projects',
-        default=_get_default_project_ids)
+        default=_get_default_project_ids,
+        help="Projects in which this stage is present. If you follow a similar workflow in several projects \n"
+            "You can share this stage among them and get consolidated information this way.")
```

Comment:
```suggestion
        help="Projects in which this stage is present. If you follow a similar workflow in several projects,\n"
            " you can share this stage among them and get consolidated information this way.")
```
I don't know if the `\n` is really needed.